<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Advance</class>
 <widget class="QMainWindow" name="Advance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Option">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>228</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Please Select Your Option:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>421</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Select</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>New Faculty Insert</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Current Faculty Present</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Current Faculty Absent</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Next Holiday declaration</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Faculty Leave Request</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Print Monthly Attendence</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Salary Calculator</string>
     </property>
    </item>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>121</y>
      <width>491</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="Id">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>231</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="name">
    <property name="geometry">
     <rect>
      <x>162</x>
      <y>120</y>
      <width>231</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="Leave_Slots">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>310</y>
      <width>240</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Select Leave Category..</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Credit leaves</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Earned Leaves</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Leave without Payment</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="EL">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Earned Leaves:</string>
    </property>
   </widget>
   <widget class="QLabel" name="CL">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Credit Leaves:</string>
    </property>
   </widget>
   <widget class="QLabel" name="LWP">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>170</y>
      <width>201</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Leave Without Payments:</string>
    </property>
   </widget>
   <widget class="QLabel" name="TL">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>220</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Total Leaves Left:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>360</y>
      <width>311</width>
      <height>78</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Refrence for leave..&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="TimeFrom">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>260</y>
      <width>77</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Time From:</string>
    </property>
   </widget>
   <widget class="QLabel" name="TimeTo">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>310</y>
      <width>57</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Time To:</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="DateTo">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>260</y>
      <width>110</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="Datefrom">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>310</y>
      <width>110</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="LeaveType">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>181</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Select Leave Type..</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Half Day</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Full Day</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="proceed">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>460</y>
      <width>231</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Proceed</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>460</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="savetofile">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>490</y>
      <width>230</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Save to .csv</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>539</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
